### DISCUSSION TITLE
How to send block to memos

### DISCUSSION THREAD

**Discussion by fordchao** [EINDEX/logseq-memos-sync#12] [Q&A] [answered]
Can send block to Memos now!

How does this work?

**Comment by EINDEX** [comment, marked as answer]
in the setting, enable some of those.
<img width="610" alt="image" src="https://github.com/EINDEX/logseq-memos-sync/assets/13883964/c7885cf9-1f05-4130-a2ac-f7482b56e468">

then you will have a slash command to send text to memos.

**Reply by fordchao** [reply]
Thanks for reply.
I have enable the setting and reopen Logseq.
After some tests, it seems that I should write something first before the slash command to send text to memos, right? Otherwise an empty memo will be send to memos immediately.
So the right way to use this function is: write something in a block, type a slash and choose the command at the very beginning of the block. Then the content of the block will be sent to memos soon.
Am I right?
